src/App.js

```diff
@@ -1,5 +1,34 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "./constants/message.js";
+import { isPlayerInputValid, isTrialInputValid } from "./utils/validation.js";
+import move from "./utils/move.js";
+import printResult from "./utils/printResult.js";
+
 class App {
-  async play() {}
+  async play() {
+    // 게임 시작
+    const playerInput = await MissionUtils.Console.readLineAsync(MESSAGES.START_INPUT);
+    const players = isPlayerInputValid(playerInput);
+    const playersMap = new Map(players.map((player) => [player, 0]));
+
+    const tryInput = await MissionUtils.Console.readLineAsync(MESSAGES.TRY_INPUT);
+    const tryNum = isTrialInputValid(tryInput);
+
+    // 게임 진행
+    MissionUtils.Console.print(MESSAGES.RESULT);
+    for (let i = 0; i < tryNum; i++) {
+      move(playersMap);
```

Comment:
```suggestion
      carsMove(playersMap);
```
`move` 라는 이름이 약간 함수 이름보다는 정적인 변수에 어울리는 것 같아요 조금 더 목적성을 드러내도 좋을 것 같아요!